yogstation/code/game/gamemodes/traitor/double_agents.dm

```diff
@@ -0,0 +1,47 @@
+/datum/game_mode/traitor/internal_affairs/make_antag_chance()
+	return FALSE
+
+/datum/antagonist/traitor/internal_affairs/iaa_process()
+	return
+
+/datum/game_mode/traitor/internal_affairs/check_finished(force_ending) //to be called by SSticker
```

Comment:
Why don't you put this in ``/datum/antagonist/traitor/internal_affairs/process()``?

Reply:
cause i'm an idiot